### PR TITLE
fix: check apisix dir is exist before clone repo

### PR DESCRIPTION
**Pull request by tzssangglass**
check apisix dir is exist before clone repo

### CODE REVIEW

**Comment by EnableAsync** [comment]
CI failed, I'm not sure exactly why, it looks like APISIX needs pcre, should we install pcre?

PS: After I tried adding pcre[1], it packaged correctly[2].

[1] https://github.com/EnableAsync/apisix-build-tools/blob/master/utils/install-common.sh#L59
[2] https://github.com/EnableAsync/apisix-build-tools/runs/3279303504?check_suite_focus=true

**Comment by tzssangglass** [comment]
> CI failed, I'm not sure exactly why, it looks like APISIX needs pcre, should we install pcre?
> 
> PS: After I tried adding pcre[1], it packaged correctly[2].
> 
> [1] [https://github.com/EnableAsync/apisix-build-tools/blob/master/utils/install-common.sh#L59](https://github.com/EnableAsync/apisix-build-tools/blob/master/utils/install-common.sh?rgh-link-date=2021-08-09T10%3A36%3A25Z#L59)
> [2] [EnableAsync/apisix-build-tools/runs/3279303504?check_suite_focus=true](https://github.com/EnableAsync/apisix-build-tools/runs/3279303504?check_suite_focus=true)

yes, we should install `pcre-devel`

**Comment by tzssangglass** [comment]
@EnableAsync can you submit a PR for `pcre-devel`?

**Comment by EnableAsync** [comment]
> @EnableAsync can you submit a PR for `pcre-devel`?

OK 😊

File: utils/install-common.sh [hunk 1]
```diff
@@ -56,8 +56,7 @@ install_openresty_deb() {
 install_openresty_rpm() {
     # install openresty and openssl111
     yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    yum install -y pcre-devel
-    yum install -y openresty openresty-openssl111-devel
+    yum install -y openresty openresty-openssl111-devel pcre-devel
```
Comment (spacewander) [inline utils/install-common.sh:59]: We should also add pcre itself as the dependency for the fpm package. It can be done in the next PR.

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
It is quite interesting that we don't need to update the deb script to make CI pass? Doesn't the deb package also require the libpcre3-dev?

**Comment by EnableAsync** [comment]
> It is quite interesting that we don't need to update the deb script to make CI pass? Doesn't the deb package also require the libpcre3-dev?

When I added deb support, I found out that openresty depends on pcre[1], so I installed it.

For the rpm I just copied the original code, so pcre was not installed.

[1] http://openresty.org/en/installation.html#debian-and-ubuntu-users
[2] https://github.com/api7/apisix-build-tools/blob/master/utils/install-common.sh#L48

**Comment by spacewander** [comment]
So maybe something in the build environment already installed the package.

**Pull request merged**